src/Domain/Infra/DependencyInjection/BundleHelper.php

```diff
@@ -208,11 +208,17 @@ private static function initDoctrineOrm(ContainerBuilder $container): void
             ->addTag('doctrine.event_listener', ['event' => DoctrineOrmEvents::loadClassMetadata])
             ->addTag('msgphp.domain.process_class_mapping', ['argument' => '$mappings']);
 
+        $container->register(DoctrineInfra\MappingConfig::class)
+            ->setPublic(false)
+            ->setArgument('$keyMaxLength', '%msgphp.doctrine.mapping.key_max_length%');
+
         if (FeatureDetection::hasFrameworkBundle($container)) {
             $container->register(DoctrineInfra\MappingCacheWarmer::class)
                 ->setPublic(false)
                 ->setArgument('$dirName', 'msgphp/doctrine-mapping')
                 ->setArgument('$mappingFiles', '%msgphp.doctrine.mapping_files%')
+                ->setArgument('$configDir', '%msgphp.doctrine.config_folder%')
```

Comment:
i like to put these as well in the MappingConfig ($mappingFiles and $mappingDir)